setup.cfg

```diff
@@ -4,18 +4,27 @@ style = pep440
 versionfile_source = src/_version.py
 versionfile_build = src/_version.py
 tag_prefix =
+parentdir_prefix = python-ci-template
 
 [tool:pytest]
 addopts = --doctest-modules --cov=./tests
```

Comment:
should be --cov=./src

Reply:
Yup you are right... dunno why it had always been like that... I'll fix it!